nest/02.introduction.md

```diff
@@ -0,0 +1,51 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+A package is a compressed file archive containing all the files required to install, uninstall and update a particular software or application.
+
+A package manager allows the user to install, upgrade or remove software packages on their OS, in other words, to **manage packages**.
+It is similar to an app store for a smartphone, or the Windows Store for example.
+One of the key differences between those and Nest (and most package managers), is that the upgrading must be done manually.
+On a smartphone, the applications are usually automatically upgraded.
+With Nest, it's up to the user to upgrade their installed packages whenever they want.
+
+Currently, to use Nest, one must use its Command Line Interface (CLI).
+This might sound intimidating for someone who has never seen a terminal or typed any command before, but if that's the case, don't worry! This guide is here to help you feel at home.
+
+## 1.2 Packages and dependencies
+It is common for a package to require other ones to work properly.
+For example, let's take a software that allows its users to send messages to each other, over the Internet.
+That software will need a way to send messages over the Internet.
+To do so, it will either implement its own solution, or use an existing library for that purpose.
+In the latter case, the software **requires** that library, because without it, the messages simply can't be sent.
+In other words, that library is a **dependency** of the software.
+
+Whenever you install a package via a package manager (and Nest as well), it will check if the package in question has any dependencies.
+If it does, Nest will install the dependencies that aren't already installed.
+So, when you install a package, don't be surprised if you see other packages getting installed too, they are just dependencies needed for that package to work properly.
+
+## 1.3 Repositories
+Packages are stored in repositories.
+Within each repository, there are a certain number of categories, and a package has to belong to one, and only one of them.
+The categories are a way to distinguish packages with the same name, but having different purposes.
+This means that we can have two packages with the same name, in the same repository but not in the same category, coexisting together, without any issue.
```

Comment:
```suggestion
This means two packages with the same name and in the same repository but not in the same category can coexist without any issue.
```
Again, this seems to be stated as a requirement.

Reply:
your sentence gets a bit long without any comma, how does this sound?
```suggestion
This means that two packages with the same name and in the same repository, but not in the same category, can coexist without any issue.
```

Reply:
I'd argue the opposite: you use quite a lot of commas.
I don't think they help with the legibility of the sentence. I find the one before "can coexist" weird, and removing it makes the previous one quite weird too. I'd ask a native English speaker but I don't have one around, so I'll let you decide what you prefer.